src/pages/Tos/Tos.test.js

```diff
@@ -0,0 +1,7 @@
+// import React from 'react'
+// import Settings from './Settings'
+// import { render } from '@testing-library/react'
+
+describe('Settings', () => {
```

Comment:
not that it matters, but should be 'Tos'

Reply:
Got it and I'll introduce a baseline test here as well.